Integrations/ARIA/aria_packet_intelligence_description.html

```diff
@@ -0,0 +1,4850 @@
+<p>
+The ARIA Cybesecurity Solutions Software-Defined Security (SDS)  platform integrates with Demisto to add robustness when responding to incidents. The combination of ARIA hardware, in the form of a Secure Intelligent Adapter (SIA), and software, specifically Packet Intelligence and SDS orchestrator (SDSo), provides the elements required to react instantly when an incident is detected. When integrated with the ARIA solution, you can create playbooks that instruct one or more SIAs to add, modify, or delete rules automatically. These rule changes, which take effect immediately, can block conversations, redirect packets to a recorder or VLAN, or perform a variety of other actions. 
```

Comment:
Please replace the README with this

Reply:
There should be on `README.md` file, which contains the HTML documentation and not the markdown documentation